ui/src/app/machines/views/MachineDetails/MachineConfiguration/PowerForm/PowerForm.test.tsx

```diff
@@ -1,11 +1,13 @@
-import { mount } from "enzyme";
```

Comment:
I've removed a few tests in this file that will become obsolete once the follow-up read-only branch lands - I didn't want to rewrite enzyme tests to RTL only to have to remove them straight away